lib/linguist/languages.yml

```diff
@@ -6184,7 +6184,19 @@ Scilab:
   - ".tst"
   tm_scope: source.scilab
   ace_mode: text
-  language_id: 344
+  language_id: 344  
+Scpel:
+  type: programming
+  color: "#650791"
+  extensions:
+  - ".co"
+  - ".ho"
+  - ".h"
```

Comment:
This extension is shared with three other C-family language so a heuristic _will_ be required for this specific use.

The heuristic is [here](https://github.com/github/linguist/blob/c1c34e5260797b4d598f5ec76f19723bfc5a1894/lib/linguist/heuristics.yml#L311-L317) and any additions need to meet the explanation in the docs [here](https://github.com/github/linguist/blob/master/docs/troubleshooting.md#my-ccobjective-c-h-header-file-is-detected-as-the-wrong-language)... ie, the heuristic must match this language and only this language, _after_ all the other more popular users.